tcms/issuetracker/types.py

```diff
@@ -383,6 +384,87 @@ def report_issue_from_testcase(self, caserun):
 
         return url + '/issues/new?' + urlencode(args, True)
 
+class REDMINE(IssueTrackerType):
+    """
+        Support for Redmine. Requires:
+
+        :api_url: - the API URL for your Redmine instance
+        :api_password: - Redmine API Key
+
+        Additional control can be applied via the ``JIRA_OPTIONS`` configuration
+        setting (in ``product.py``). By default this setting is not provided and
+        the code uses ``jira.JIRA.DEFAULT_OPTIONS`` from the ``jira`` Python module!
```

Comment:
This paragraph is invalid, copy-paste error. Update it or remove it.

Reply:
Agree, will remove it.